packages/renderer/src/lib/help/HelpPage.svelte

```diff
@@ -1,91 +1,187 @@
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
 
-      <li class="pb-2">
-        <i class="fas fa-solid fa-comment" aria-hidden="true"></i>
-        <p
-          on:click="{() =>
-            window.openExternal('https://github.com/containers/podman-desktop/discussions/categories/general')}"
-          title="https://github.com/containers/podman-desktop/discussions/categories/general"
-          class="text-sm inline-flex ml-1 cursor-pointer text-violet-400 hover:text-violet-600 hover:no-underline">
-          Create a GitHub discussion
-        </p>
-      </li>
+      <!-- Core Podman Engine -->
+      <div class="bg-zinc-800 px-3 pt-3 pb-3 rounded-lg">
+        <div class="text-lg">Core Podman Engine</div>
+        <div class="flex sm:flex-col md:flex-row py-3 md:h-24">
+          <div class="sm:h-full sm:my-auto md:flex md:flex-col md:w-1/2 md:border-r-2">
+            <div class="sm:mt-2 md:my-auto">
+              <p
+                class="generic-link text-gray-100 cursor-pointer hover:no-underline"
+                on:click="{() => window.openExternal('https://podman.io/getting-started/')}"
+                title="https://podman.io/getting-started/">
+                Getting started with containers<i class="fas fa-solid fa-external-link-alt ml-2" aria-hidden="true"></i>
+              </p>
+            </div>
+            <div class="sm:mt-2 md:my-auto">
+              <p
+                class="generic-link text-gray-100 cursor-pointer hover:no-underline"
+                on:click="{() => window.openExternal('https://docs.podman.io/en/latest/Commands.html')}"
+                title="https://docs.podman.io/en/latest/Commands.html">
+                View podman commands<i class="fas fa-solid fa-external-link-alt ml-2" aria-hidden="true"></i>
+              </p>
+            </div>
+          </div>
+          <div class="md:flex md:flex-col md:w-1/2">
+            <div class="sm:mt-2 md:my-auto md:text-right">
+              <p
+                class="generic-link text-gray-100 cursor-pointer hover:no-underline"
+                on:click="{() => window.openExternal('https://podman.io/getting-started/installation')}"
+                title="https://podman.io/getting-started/installation">
+                Set up podman<i class="fas fa-solid fa-external-link-alt ml-2" aria-hidden="true"></i>
+              </p>
+            </div>
+            <div class="sm:mt-2 md:my-auto md:text-right">
+              <p
+                class="generic-link text-gray-100 cursor-pointer hover:no-underline"
+                on:click="{() => window.openExternal('https://docs.podman.io/en/latest/Tutorials.html')}"
+                title="https://docs.podman.io/en/latest/Tutorials.html">
+                View all tutorials<i class="fas fa-solid fa-external-link-alt ml-2" aria-hidden="true"></i>
+              </p>
+            </div>
+          </div>
+        </div>
```

Comment:
These links are supposed to be brought by the extensions so it can't be hardcoded.
(it's like the link that we see on each provider)

Reply:
updated, now links fetches from the contributions